### ISSUE TITLE
VIM highlighting

### ISSUE THREAD

**Issue by hellerve** [tj/luna#75]
Hey,

I've started a [vim plugin](https://github.com/hellerve/luna-vim). Up until now, it's basically a trimmed-down version of vim-ruby but with type definition blocks. It is laggy and makes a load of mistakes when hightlighting (especially with type annotations _shudder_), but to an extent, it works.

I've been having trouble with the minimalism of the examples, though, and the lack of documentation on the available operators. I think I've pulled them all out of the parser file (thanks for the Backus-Naur style comments, by the way), but you never know.

Anyway, if anyone who knows Luna and/or VimScript better than I do could take a look at it, that would be great!

Cheers

**Comment by vendethiel** [comment]
I gather you've taken a look at the examples already?

**Comment by hellerve** [comment]
Yes, I did. As I said they are rather small and the operators are not documented or showcased in there, I think.

**Comment by glhrmfrts** [comment]
I'm afraid what you are looking for will not be available any time soon, Luna still has a lot of undefined/unclear features.

**Comment by hellerve** [comment]
Reasonable. Just hit me up whenever y'all feel the language is stable enough syntax-wise and I will gladly maintain a VIM plugin.
